xml/dhcp_vendor_class.xml

```diff
@@ -305,4 +359,40 @@
     <param pos="0" name="os.vendor" value="Cisco"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^BrightSign ((?:HS|XD|LS|XT|HD)\d{3,4}(?:-W|-PP)?)$">
+    <description>BrightSign Digital Signage Player</description>
+    <example hw.model="HS124">BrightSign HS124</example>
+    <example hw.model="XD1034">BrightSign XD1034</example>
+    <example hw.model="XD1034-W">BrightSign XD1034-W</example>
+    <example hw.model="LS424-W">BrightSign LS424-W</example>
+    <example hw.model="XT244">BrightSign XT244</example>
+    <example hw.model="HD1023">BrightSign HD1023</example>
+    <example hw.model="XT1144-PP">BrightSign XT1144-PP</example>
+    <param pos="0" name="hw.vendor" value="BrightSign"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
```

Comment:
Would "Media Player" be a more appropriate value for `*.device` than "Media Server"?
> BrightSign LLC, the global market leader in digital signage media players ...

Reply:
I went with Media Sever as both favicons.xml and html_title.xml databases have existing entries for BrightSign that use Media Server for hw.device.
(while also referring to the device as a Media Player in description and hw.product ?)

Reply:
Thank you for pointing those out as I overlooked them. Then we'll accept this as-is and change them all in a separate PR if appropriate.